packages/bpk-component-nudger/src/BpkNudger.tsx

```diff
@@ -59,24 +59,21 @@ const BpkNudger = ({
           >
             <BpkText textStyle={TEXT_STYLES.heading5}>{title}</BpkText>
             {subtitle && (
-              // TODO: className to be removed
-              // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-              <BpkText className={getClassName('bpk-nudger__label--subtitle')}>
-                {subtitle}
-              </BpkText>
+              <span className={getClassName('bpk-nudger__label--subtitle')}>
+                <BpkText>
+                  {subtitle}
+                </BpkText>
+              </span>
             )}
           </span>
-        </BpkLabel>
+        </div>
       )}
       <BpkConfigurableNudger
         inputClassName={getClassName('bpk-nudger__input--numeric')}
         compareValues={compareValues}
         incrementValue={incrementValue}
         decrementValue={decrementValue}
         formatValue={formatValue}
-        // TODO: className to be removed
-        // eslint-disable-next-line @skyscanner/rules/forbid-component-props
-        className={className}
```

Comment:
If we are removing this here should we also remove it from the API and also this PR would then become a major as we are removing functionality that folks could be using?

Reply:
Agreed. I don't think we can remove these "consumer pass-through" `className` cases. Tbh quite unsure on how to tackle these long term. As you say Ollie they can only be removed through a breaking change.

Reply:
Ah sorry, my bad, I got confused about this failure: https://github.com/Skyscanner/backpack/actions/runs/8649230361/job/23715083356#step:8:134
```
/home/runner/work/backpack/backpack/packages/bpk-component-nudger/src/BpkNudger.tsx
Error:   77:9  error  Prop className is forbidden. See ./decisions/className.md  @skyscanner/rules/forbid-component-props
```

Reply:
Do I just need to disable the linter for the line? @olliecurtis @mungodewar

Reply:
Or do I add it to `.eslintrc.js` in an `allowedFor` rules?

Reply:
Let's leave it inline till we work out what is to be done with these cases.